conf/layer.conf

```diff
@@ -0,0 +1,15 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "labgrid"
+BBFILE_PATTERN_labgrid = "^${LAYERDIR}/"
+BBFILE_PRIORITY_labgrid = "6"
+
```

Comment:
The priority should be larger that the priority in meta-python otherwise your recipes overriding the recipes in meta-python is not picked up.
This is as an example a problem for the python3-multidict recipe.

Reply:
Set priority to one higher than meta-python